numpy/core/_type_aliases.pyi

```diff
@@ -0,0 +1,19 @@
+import sys
+from typing import Dict, Union, Type, List
+
+from numpy import generic, signedinteger, unsignedinteger, floating, complexfloating
+
+if sys.version_info >= (3, 8):
+    from typing import TypedDict
+else:
+    from typing_extensions import TypedDict
+
+class _SCTypes(TypedDict):
+    int: List[Type[signedinteger]]
+    uint: List[Type[unsignedinteger]]
+    float: List[Type[floating]]
+    complex: List[Type[complexfloating]]
+    others: List[type]
```

Comment:
```suggestion
    others: List[Type[generic]]
```
I think is accurate?

Reply:
Nope, I'm wrong. Seems like a good argument to never use `sctypes` to me, but that's out of scope for this PR...